extras/dialer/time-scope-probe-cgroup

```diff
@@ -28,7 +28,7 @@ TIME_K2=$(awk '/system/ {print $2}' "${cpuacct_stat}")
 TIME_T2=$((TIME_U2 + TIME_K2))
 
 echo "utime;stime;total_time"
-for ((i = 0; i < "$interval_num"; i++)); do
+for ((i = 0; i < interval_num; i++)); do
```

Comment:
Apparently, with the double-parenthesis construct, no need for the `$` prefix on variables, or quotes. @schu told me so today.

http://tldp.org/LDP/abs/html/dblparens.html